backend/src/resolvers/users.js

```diff
@@ -0,0 +1,15 @@
+import { neo4jgraphql } from 'neo4j-graphql-js'
+import fileUpload from './fileUpload'
+
+export default {
+  Mutation: {
+    UpdateUser: async (object, params, context, resolveInfo) => {
+      params = await fileUpload(params, { file: 'avatarUpload', url: 'avatar' })
+      return neo4jgraphql(object, params, context, resolveInfo, false)
+    },
+    CreateUser: async (object, params, context, resolveInfo) => {
+      params = await fileUpload(params, { file: 'avatarUpload', url: 'avatar' })
+      return neo4jgraphql(object, params, context, resolveInfo, false)
+    },
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-05-21T23:00:36Z" title="Wednesday, May 22nd 2019, 1:00:36 am +02:00">May 22, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,15 @@
+import { neo4jgraphql } from 'neo4j-graphql-js'
+import fileUpload from './fileUpload'
+
+export default {
+  Mutation: {
+    UpdateUser: async (object, params, context, resolveInfo) => {
+      params = await fileUpload(params, { file: 'avatarUpload', url: 'avatar' })
+      return neo4jgraphql(object, params, context, resolveInfo, false)
+    },
+    CreateUser: async (object, params, context, resolveInfo) => {
+      params = await fileUpload(params, { file: 'avatarUpload', url: 'avatar' })
+      return neo4jgraphql(object, params, context, resolveInfo, false)
+    }
```


Love :heart_eyes_cat: how clean this code is!!